neo.UnitTests/UT_ProtocolHandlerMailbox.cs

```diff
@@ -0,0 +1,194 @@
+﻿using Akka.TestKit;
+using Akka.TestKit.Xunit2;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using Moq;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Network.P2P;
+using Akka.Configuration;
+using Neo.IO;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Neo.UnitTests
+{
+    [TestClass]
+    public class UT_ProtocolHandlerMailbox  : TestKit
+    {
+        private static readonly Random TestRandom = new Random(1337); // use fixed seed for guaranteed determinism
+
+        ProtocolHandlerMailbox uut;
+
+        [TestCleanup]
+        public void Cleanup()
+        {
+            Shutdown();
+        }
+
+        [TestInitialize]
+        public void TestSetup()
+        {
+            Akka.Actor.ActorSystem system = Sys;
+            var config = TestKit.DefaultConfig;
+            var akkaSettings = new Akka.Actor.Settings(system, config);
+            uut = new ProtocolHandlerMailbox(akkaSettings, config);
+        }
+
+        [TestMethod]
+        public void ProtocolHandlerMailbox_Test_IsHighPriority()
+        {
+            ISerializable s = null;
+
+            //handshaking
+            uut.IsHighPriority(Message.Create(MessageCommand.Version, s)).Should().Be(true);
+            uut.IsHighPriority(Message.Create(MessageCommand.Verack, s)).Should().Be(true);
+
+            //connectivity
+            uut.IsHighPriority(Message.Create(MessageCommand.GetAddr, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Addr, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Ping, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Pong, s)).Should().Be(false);
+
+            //synchronization
+            uut.IsHighPriority(Message.Create(MessageCommand.GetHeaders, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Headers, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.GetBlocks, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Mempool, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Inv, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.GetData, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.NotFound, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Transaction, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Block, s)).Should().Be(false);
+            uut.IsHighPriority(Message.Create(MessageCommand.Consensus, s)).Should().Be(true);
+            uut.IsHighPriority(Message.Create(MessageCommand.Reject, s)).Should().Be(false);
```

Comment:
Here the tests @shargon! hahaha 

Focused on ProtocolHandler this time... Coverage increased a lot already, and on future we cover others ;)